src/extensions/extensionsApi.ts

```diff
@@ -320,20 +320,20 @@ export async function listInstances(projectId: string): Promise<ExtensionInstanc
  * @param params params to configure the extension instance
  * @param validateOnly if true, only validates the update and makes no changes
  */
-export async function configureInstance(
-  projectId: string,
-  instanceId: string,
-  params: { [option: string]: string },
-  validateOnly: boolean = false
-): Promise<any> {
+export async function configureInstance(args: {
+  projectId: string;
+  instanceId: string;
+  params: { [option: string]: string };
+  validateOnly?: boolean;
```

Comment:
nit: boolean > boolean|null 

consider giving this a default of `false` instead

Reply:
I wanted to, but unfortunately you can't use default values inside of an args object like this. I figured that it was preferable to have named params here instead of a little extra type safety